.ci/azure/linux_debian.yml

```diff
@@ -242,9 +243,12 @@ jobs:
       export DATA_PATH=$(MODELS_PATH)
       export MODELS_PATH=$(MODELS_PATH)
       # for libopenvino_template_extension.so
-      export LD_LIBRARY_PATH=$(INSTALL_TEST_DIR):$LD_LIBRARY_PATH
-      export PYTHONPATH=$(INSTALL_TEST_DIR):$(INSTALL_DIR)/python/python3.8:$PYTHONPATH
-      export OV_FRONTEND_PATH=$(INSTALL_TEST_DIR)
+      export LD_LIBRARY_PATH=$(INSTALL_TEST_DIR):$(INSTALL_DIR)/python/python3.8:$LD_LIBRARY_PATH
+      export OV_FRONTEND_PATH=$(PYTHON_WHEEL_INSTALL_DIR)/openvino/libs:$(INSTALL_TEST_DIR)
+      export PYTHONPATH=$(INSTALL_TEST_DIR)
```

Comment:
it would be nice to add comments what paths are needed for what

Reply:
Yes, when figure out what has to be set to run these python mock tests then will add comments